configurations-conception-communes/fil-ariane.md

```diff
@@ -0,0 +1,243 @@
+---
+altLangPage: "https://design.canada.ca/common-design-patterns/breadcrumb-trail.html"
+breadcrumbs:
+  - title: "À propos de Canada.ca"
+    link:  "https://www.canada.ca/fr/gouvernement/a-propos.html"
+  - title: "Système de conception de Canada.ca"
+    link: "https://www.canada.ca/fr/gouvernement/a-propos/systeme-conception.html"
+  - title: "Bibliothèque de modèles et de configurations de conception"
+    link: "https://conception.canada.ca/configurations-conception-communes/connexion-contextuel.html"
+  - title: "En-tête global"
+    link: "https://conception.canada.ca/configurations-conception-communes/en-tete-general.html"	
```

Comment:
1/3 We could move this file into a folder "connexion" and then apply a default breadcrumb in the _config.yml

Reply:
One of the goals of the migration is to flatten our files, so I'm not sure a mini-directory is a good solution.  Let's chat about this.  I do get putting "root" breadcrumbs in the config file, however.

Reply:
We also do not want to change the path or file names because we do not have a proper redirect strategy in place.